app/templates/main/browse-transferring-body.html

```diff
@@ -319,22 +336,39 @@ <h3 class="govuk-heading-s govuk-heading-s--browse-all-filter">Date consignment
     </div>
     <!-- MOBILE SORT -->
     <div class="govuk-form-group sort-container__form mobile-sort">
-        <div class="browse__sort-container">
-            <label class="govuk-label" for="sort">Sort by</label>
-            <select class="govuk-select govuk-select__sort-container-select"
-                    id="sort"
-                    name="sort">
-                <option value="body-a">Transferring body (A to Z)</option>
-                <option value="body-b" selected>Transferring body (Z to A)</option>
-                <option value="series-a">Series (A to Z)</option>
-                <option value="series-b">Series (Z to A)</option>
-                <option value="date-first">Date record transferred (most recent first)</option>
-                <option value="date-last">Date record transferred (oldest first)</option>
-            </select>
-            <button class="govuk-button govuk-button__sort-container-update-button"
-                    id="sort-update-button"
-                    data-module="govuk-button">Apply</button>
-        </div>
+        <form action="{{ url_for('main.browse') }}" method="get">
```

Comment:
the mobile and desktop  forms and all nested html looks very similar. Are we able to extract the shared parts to reduce duplication due to the mobile view. Maybe Im missing something that differs though. @colinbowen thoughts?

Reply:
spoke on offline, going to be dealt with in a refactor PR